### PR TITLE
Advise running ntpd on Brooklyn server

### PR DESCRIPTION
**Pull request by aledsage**

### CODE REVIEW

File: guide/ops/troubleshooting/deployment.md [hunk 1]
```diff
@@ -40,9 +40,12 @@ means there was some problem obtaining or connecting to the machine.
 An error like `... Not authorized to access cloud ...` usually means the wrong identity/credential was used.
 
 AWS requires a X-Amz-Date header which contains the date of the Apache Brooklyn AWS client.
-If the date on the server is wrong, for example several minutes behind you will get Authorization Exception.
-Please be sure that the machine which is running Apache Brooklyn set its clock correctly.
-To set the time on Linux we advice to use the ntp client: `sudo ntpdate pool.ntp.org`.
+If the date on the server is wrong, for example several minutes behind you will get Authorization 
+Exception. This is prevent replay attacks. Please be sure that the machine which is running
```
Comment (alasdairhodge) [inline guide/ops/troubleshooting/deployment.md]: Missing "to"

**Comment by alasdairhodge** [comment]
One tiny typo, but otherwise fine and dandy.

File: guide/ops/troubleshooting/deployment.md [hunk 2]
```diff
@@ -40,9 +40,12 @@ means there was some problem obtaining or connecting to the machine.
 An error like `... Not authorized to access cloud ...` usually means the wrong identity/credential was used.
 
 AWS requires a X-Amz-Date header which contains the date of the Apache Brooklyn AWS client.
-If the date on the server is wrong, for example several minutes behind you will get Authorization Exception.
-Please be sure that the machine which is running Apache Brooklyn set its clock correctly.
-To set the time on Linux we advice to use the ntp client: `sudo ntpdate pool.ntp.org`.
+If the date on the server is wrong, for example several minutes behind you will get Authorization 
+Exception. This is prevent replay attacks. Please be sure that the machine which is running  
+Apache Brooklyn set its clock correctly. To set the time on Linux you can use the ntp client 
+(e.g. `sudo ntpdate pool.ntp.org`). We advice running the
```
Comment (sjcorbett) [inline guide/ops/troubleshooting/deployment.md]: typo: advice

**Comment by aledsage** [comment]
Thanks; incorporated fixes - merging now.

**Pull request merged**